src/compat/floret/io.rs

```diff
@@ -106,10 +108,52 @@ impl ReadFloretText for Embeddings<FloretSubwordVocab, NdArray> {
     }
 }
 
+/// Write embeddings in the floret format.
+pub trait WriteFloretText
+where
+    Self: Sized,
+{
+    /// Read embeddings in the floret format.
+    fn write_floret_text(&self, write: &mut dyn Write) -> Result<()>;
+}
+
+impl WriteFloretText for Embeddings<FloretSubwordVocab, NdArray> {
```

Comment:
I haven't looked into this, so this is just a genuine question: Are there compatibility issues between our quantized storage interface and floret embeddings?

Reply:
I don't think so. Since the quantized storage is oblivious to the vocabulary, it should just work if we have a Vocab chunk for floret in place.

Would be cool, a small embedding matrix thanks to bloom embeddings, compressed even further through quantization.